### PR TITLE
Windows app cloud hardening

### PR DESCRIPTION
**Pull request by krystian-panek-wttech**
much more hardened disp setup for AEM SDK

### CODE REVIEW

File: pkg/project/app_cloud/Taskfile.yml [hunk 1]
```diff
@@ -203,9 +203,12 @@ tasks:
     aliases: [dispatcher:up]
     dir: dispatcher
     cmds:
-      # - sh aem/home/opt/sdk/dispatcher/bin/validate.sh dispatcher/src > aem/home/var/log/dispatcher-validate.log # uncomment if possible, see: https://github.com/adobe/aem-project-archetype/issues/1043
+      - 'command -v docker || { echo "Docker is not installed! Visit: https://www.docker.com/products/docker-desktop"; exit 1; }'
+      - 'test -d ../aem/home/opt/sdk/dispatcher || { echo "Dispatcher SDK is not available. Try providing AEM SDK ZIP instead of JAR to directory: aem/home/lib"; exit 1; }'
```
Comment (krystian-panek-wttech) [inline pkg/project/app_cloud/Taskfile.yml:207]: fixes #94

File: pkg/instance_manager.go [hunk 2]
```diff
@@ -272,13 +273,17 @@ func InstanceIds(instances []Instance) string {
 	return strings.Join(lo.Map(instances, func(i Instance, _ int) string { return i.id }), ",")
 }
 
-func InstanceMsg(instances []Instance, msg string) string {
+func InstanceMsg(instance Instance, msg any) string {
+	return stringsx.AddPrefix(fmt.Sprintf("%v", msg), fmt.Sprintf("%s > ", instance.ID()))
```
Comment (krystian-panek-wttech) [inline pkg/instance_manager.go:277]: fixes #116

File: pkg/project/app_cloud/Taskfile.yml [hunk 3]
```diff
@@ -203,9 +203,12 @@ tasks:
     aliases: [dispatcher:up]
     dir: dispatcher
     cmds:
-      # - sh aem/home/opt/sdk/dispatcher/bin/validate.sh dispatcher/src > aem/home/var/log/dispatcher-validate.log # uncomment if possible, see: https://github.com/adobe/aem-project-archetype/issues/1043
-      - rm -fr target/sdk && cp -r ../aem/home/opt/sdk/dispatcher target/sdk
+      - 'command -v docker || { echo "Docker is not installed! Visit: https://www.docker.com/products/docker-desktop"; exit 1; }'
+      - 'test -d ../aem/home/opt/sdk/dispatcher || { echo "Dispatcher SDK is not available. Try providing AEM SDK ZIP instead of JAR to directory: aem/home/lib"; exit 1; }'
+      - test -d target/sdk || rm -fr target/sdk && cp -r ../aem/home/opt/sdk/dispatcher target/sdk
```
Comment (krystian-panek-wttech) [inline pkg/project/app_cloud/Taskfile.yml:208]: removing it every time was causing a runtime problem of container using volume as right after that command docker-compose wasn't redeploying container but its volume files are gone; this makes it done once

File: pkg/project/app_cloud/Taskfile.yml [hunk 4]
```diff
@@ -203,9 +203,12 @@ tasks:
...
+      # - sh target/sdk/bin/validate.sh src > ../aem/home/var/log/dispatcher-validate.log
```
Comment (krystian-panek-wttech) [inline pkg/project/app_cloud/Taskfile.yml:209]: surprisingly I have no `validate.sh` unpacked / is is not included anymore in SDK zip?  I don't know.. Adobe missed that?

**Review by jaroslaw-pietraszek-wttech — APPROVED** [review]

**Pull request merged**